library-and-framework-support-schema.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Maybe we should rename the file to `library-and-framework-list-schema.json` or `library-and-framework-overview-schema.json`. @vjovanov @wirthi?

Reply:
LGTM

Reply:
or `tested-libraries-schema.json`?